src/components/screens/ChapterScreen/Pagination.stories.js

```diff
@@ -1,14 +1,20 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
 import Pagination from './Pagination';
 
-const nextEntry = {
-  description: 'The description of the next chapter',
-  slug: '/slug',
-  title: 'Chapter Title',
+export default {
+  component: Pagination,
+  title: 'Screens/ChapterScreen/Pagination',
 };
 
-storiesOf('Screens|ChapterScreen/Pagination', module)
-  .addParameters({ component: Pagination })
-  .add('without nextEntry', () => <Pagination />)
-  .add('with nextEntry', () => <Pagination nextEntry={nextEntry} />);
+export const Basic = args => <Pagination {...args} />;
+Basic.storyName = 'without nextEntry';
+
+export const WithNextEntry = Basic.bind();
```

Comment:
We're recommending the following syntax:

```js
const PaginationStory = (args) => <Pagination {...args} />;

export const Basic = PaginationStory.bind({});
Basic.args = { ... };

export const WithNextEntry = PaginationStory.bind({});
WithNextEntry.args = { ... };
```

Reply:
This was fixed in a prior commit. Thanks for the time reviewing it.